routers/user/home.go

```diff
@@ -449,7 +450,6 @@ func Issues(ctx *context.Context) {
 	}
 
 	opts := &models.IssuesOptions{
-		IsClosed: util.OptionalBoolOf(isShowClosed),
```

Comment:
Why the `IsClosed` flag needs to be set later instead of here?

Reply:
If I set the isClosed flag here the search to get the issue ids will reflect that flag, which is a problem since I wanted to show an accurate count on the open and closed tabs. By not setting the flag here, the issue ids search by keyword will return all the matching issues ids and those issue ids can be fed into the calls to produce the stats that show up on the screen.